app/src/layouts/common/appSidebar/helpAndService/FAQcontent/FAQContent.jsx

```diff
@@ -0,0 +1,136 @@
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage, useIntl } from 'react-intl';
+import OpenIcon from 'common/img/open-in-new-tab-inline.svg';
+import React from 'react';
+import { referenceDictionary, uniqueId } from 'common/utils';
+import { showModalAction } from 'controllers/modal';
+import { useDispatch, useSelector } from 'react-redux';
+import { userIdSelector } from 'controllers/user';
+import { setFAQOpenStatusTrue } from 'controllers/log/storageUtils';
+import { messages } from 'layouts/common/appSidebar/helpAndService/messages';
+import { LinkItem } from 'layouts/common/appSidebar/helpAndService/linkItem/linkItem';
+import { referenceFAQDictionary } from 'common/utils/referenceDictionary';
+import styles from './FAQContent.scss';
+
+const cx = classNames.bind(styles);
+
+export const FAQContent = ({ onFAQOpen, closeNavbar, closePopover }) => {
+  const { formatMessage } = useIntl();
+  onFAQOpen(true);
+  const userId = useSelector(userIdSelector);
+  setFAQOpenStatusTrue(userId);
+  const dispatch = useDispatch();
+
+  const FAQContentItems = [
+    {
+      linkTo: referenceFAQDictionary.configureReportingFAQ,
+      message: messages.configureReporting,
+    },
+    {
+      linkTo: referenceFAQDictionary.improvePerformanceFAQ,
+      message: messages.improvePerformance,
+    },
+    {
+      linkTo: referenceFAQDictionary.configureAnalyzerFAQ,
+      message: messages.configureAnalyzer,
+    },
+    {
+      linkTo: referenceFAQDictionary.integrateJiraFAQ,
+      message: messages.integrateJira,
+    },
+    {
+      linkTo: referenceFAQDictionary.configureCertificateFAQ,
+      message: messages.configureCertificate,
+    },
+    {
+      linkTo: referenceFAQDictionary.fileStorageOptionsFAQ,
+      message: messages.fileStorageOptions,
+    },
+    {
+      linkTo: referenceFAQDictionary.pricingOptionsFAQ,
+      message: messages.pricingOptions,
+    },
+  ];
+  const openModal = () => {
+    closePopover();
+    closeNavbar();
+    dispatch(
+      showModalAction({
+        id: 'requestSupportModal',
+      }),
+    );
+  };
+
+  const furtherAssistanceValues = {
+    support: (
+      <LinkItem
+        link={referenceDictionary.rpEmail}
+        content={formatMessage(messages.supportTeam)}
+        icon={OpenIcon}
+        className={cx('inline-ref')}
+      />
+    ),
+    channel: (
+      <LinkItem
+        link={referenceDictionary.rpSlack}
+        content={formatMessage(messages.slackChannel)}
+        className={cx('inline-ref')}
+        icon={OpenIcon}
+      />
+    ),
+  };
+  return (
+    <>
+      {FAQContentItems.map((contentItem) => {
+        return (
+          <LinkItem
+            link={contentItem.linkTo}
+            content={formatMessage(contentItem.message)}
+            className={cx('menu-item')}
+            onClick={() => {
+              closePopover();
+              closeNavbar();
+            }}
+            key={`faq-${uniqueId()}`}
```

Comment:
Looks like the `key` will have different value every render that will force the React to unmount/mount it.
Can we use `linkTo` as a `key`?